client/pkg/fileutil/lock_windows.go

```diff
@@ -89,10 +86,8 @@ func lockFile(fd windows.Handle, flags uint32) error {
 	err := windows.LockFileEx(fd, flags|windows.LOCKFILE_EXCLUSIVE_LOCK, 0, 1, 0, &windows.Overlapped{})
 	if err == nil {
 		return nil
-	} else if err.Error() == errLocked.Error() {
+	} else if err == windows.ERROR_LOCK_VIOLATION {
```

Comment:
@MIVerTFT thanks! So per some reading, I think`windows.ERROR_LOCK_VIOLATION` gives the error message, `The process cannot access the file because another process has locked a portion of the file.` - please correct me if I am wrong?  In that case, `errLocked` check seems redundant and your changes makes sense.

Reply:
Yes `windows.ERROR_LOCK_VIOLATION` gives the same error. But the comparison operator compares the values of windows error codes cause the `windows.ERROR_LOCK_VIOLATION` is a `syscall.Errno = 33`. I think it is more reliable than comparing strings.

Reply:
Thanks @MIVerTFT Let me think little more, I will be adding review comments soon.

Reply:
I do not have a windows server right now, could you try to execute the following code? 

```
func checkError() {
	var err error
	err = windows.ERROR_LOCK_VIOLATION

	fmt.Printf("windows.ERROR_LOCK_VIOLATION: %s\n", err.Error())

	fmt.Printf("err.Error() == errLocked.Error(): %t\n", err.Error() == errLocked.Error())
}
```

Reply:
I tried on windows 10 and on windows server 2019 got the same result
```
c:\1>test.exe
windows.ERROR_LOCK_VIOLATION: The process cannot access the file because another process has locked a portion of the file.
err.Error() == errLocked.Error(): true
```